### PR TITLE
feat(office365): Add Office365 Message Trace.

### PR DESCRIPTION
**Pull request by PierrickV**

### CODE REVIEW

File: Office 365/message-trace/_meta/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+uuid: 8461aabe-6eba-4044-ad7f-a0c39a2b2279
+name: Message Trace
+slug: message-trace
```
Comment (squioc) [inline Office 365/message-trace/_meta/manifest.yml:3]:
```suggestion
name: Office365 Message Trace
slug: o365-message-trace
```

File: Office 365/message-trace/_meta/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+description: >-
+  Message trace follows email messages as they travel through your Exchange Online organization. You can determine if a message was received, rejected, deferred, or delivered by the service. It also shows what actions were taken on the message before it reached its final status. You can use the information from message trace to efficiently answer user questions about what happened to messages, troubleshoot mail flow issues, and validate policy changes.
```
Comment (squioc) [inline Office 365/message-trace/_meta/manifest.yml:6]:
```suggestion
 Microsoft 365 Message trace follows email messages as they travel through your Exchange Online organization. You can determine if a message was received, rejected, deferred, or delivered by the service. It also shows what actions were taken on the message before it reached its final status. You can use the information from message trace to efficiently answer user questions about what happened to messages, troubleshoot mail flow issues, and validate policy changes.
```

File: Office 365/message-trace/ingest/parser.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      - set:
+          office365.message_trace.MessageTraceId: "{{json_event.message.MessageTraceId}}"
+          office365.message_trace.Size: "{{json_event.message.Size}}"
+          office365.message_trace.Status: "{{json_event.message.Status}}"
+          event.start: "{{parse_start_date.datetime}}"
+          event.end: "{{parse_end_date.datetime}}"
+          email.delivery_timestamp: "{{parse_received_date.datetime}}"
+          source.ip: "{{json_event.message.FromIP}}"
+          email.message_id: "{{json_event.message.MessageId}}"
+          email.sender.address: "{{json_event.message.SenderAddress}}"
```
Comment (squioc) [inline Office 365/message-trace/ingest/parser.yml:46]: According to the meaning of `SenderAddress`, maybe set this value to `email.from.address`?

**Review by squioc — COMMENTED** [review]
Nice work :+1:
Here, some comments

**Comment by PierrickV** [comment]
Hello, thank you for your review !

Please review latest commit :smile:

**Review by squioc — APPROVED** [review]

**Pull request merged**